src/main/java/org/folio/search/service/converter/preprocessor/AuthorityEventPreProcessor.java

```diff
@@ -61,15 +64,25 @@ public void init() {
    */
   @Override
   public List<ResourceEvent> process(ResourceEvent event) {
-    return event.getType() == ResourceEventType.UPDATE
-           ? getResourceEventsToUpdate(event)
-           : getResourceEvents(event, event.getType());
+    log.debug("process:: by [event: {}]", event);
+
+    if (event.getType() == ResourceEventType.UPDATE) {
+      log.debug("process:: eventType == UPDATE");
+      return getResourceEventsToUpdate(event);
+    }
+    log.debug("process:: eventType != UPDATE");
+    return getResourceEvents(event, event.getType());
```

Comment:
ResourceEvent is too big to put it in logs. Only basic info could be logged (Same for other places where big object is loggged).
You shouldn't write multiple methods for different event types, you could just put eventType into a log